spec.html

```diff
@@ -36863,11 +36865,8 @@ <h1>The %AsyncFromSyncIteratorPrototype% Object</h1>
           <h1>%AsyncFromSyncIteratorPrototype%.next ( _value_ )</h1>
           <emu-alg>
             1. Let _O_ be the *this* value.
+            1. Assert: Type(_O_) is Object and _O_ has a [[SyncIterator]] internal slot.
```

Comment:
Should this be `[[SyncIteratorRecord]]`?

Reply:
+1, let's keep the slot name the same